src/overmap_ui.cpp

```diff
@@ -601,7 +602,8 @@ static void draw_ascii(
         // Ok, we found something
         if( info ) {
             const bool explored = show_explored && overmap_buffer.is_explored( omp );
-            ter_color = explored ? c_dark_gray : info->get_color( uistate.overmap_show_land_use_codes );
+            ter_color = explored ? oter_explored.obj().get_color() :
```

Comment:
I did not profile this, but it is very likely that it would have less overhead and be more performant if explored color would be stored in variable of `nc_color` type that would be assigned only once in finalize step of overmap terrain data load (i.e. `overmap_terrains::finalize()`) instead of accessing `oter_explored.obj().get_color()` for each overmap tile.